internal/cli/kubernetes/config/apply.go

```diff
@@ -173,7 +173,7 @@ func ApplyBuilder() *cobra.Command {
 }
 
 func getOperatorMajorVersion(image string) string {
-	version := strings.Trim(image, fmt.Sprintf("%s:", containerImage))
+	version := strings.TrimPrefix(image, containerImage+":")
```

Comment:
@mongodb/atlas_kubernetes_team this was an interesting one, removing fmt flagged an incorrect usage of Trim and that we should use TrimPrefix, can you confirm?

Reply:
technically this behaves differently but indeed the `TrimPrefix` makes much more sense. LGTM from my side but leaving the review to have another set of eyes to @josvazg @helderjs @roothorp

Reply:
yeah LGTM as well, we are only interesting on removing a prefix, that should not happen also at the end.